src/drivers/sdl/lv_sdl_private.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file lv_sdl_private.h
+ *
+ */
+
+#ifndef LV_SDL_PRIVATE_H
+#define LV_SDL_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
```

Comment:
Should include "lv_conf_internal.h" firstly in order to get value of `LV_USE_SDL`.

Currently it's working because lv_sdl_private.h is included after other headers which includes lv_conf_internal.h for you.

Reply:
OK. However, since the header file contains lv_display_t, so I changed it to lv_display.h